platform/podjs/src/browserPod.ts

```diff
@@ -307,15 +307,32 @@ class IDBPolyOut implements PolyOut {
     }
 
     async stat(id: string): Promise<Stats> {
-        if (id != "") return (await this.getFile(id)).stat();
-        return {
-            getId: () => "",
-            getSize: () => 0,
-            getTime: () => "",
-            getName: () => "",
-            isFile: () => false,
-            isDirectory: () => true,
-        };
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const stats: any =
```

Comment:
Are you sure this can't be `Stats`?

Reply:
Yes, then I couldn't monkey patch the object. There's probably a more TypeScript-ish way to do this, but I've wasted enough time fighting with TypeScript and ESLint on something so trivial :D